src/controllers/reportingController.ts

```diff
@@ -1,77 +1,53 @@
-import { OrganisationalUnit } from '../../src/csrs/model/organisationalUnit'
+import {Request, Response, Router} from 'express'
+import {ReportService} from '../report-service'
+import {CsrsService} from 'src/csrs/service/csrsService'
+import {OrganisationalUnitService} from 'src/csrs/service/organisationalUnitService'
+import {OrganisationalUnit} from '../../src/csrs/model/organisationalUnit'
+import {CourseCompletionsSession} from './reporting/model/courseCompletionsSession'
+import {fetchCourseCompletionSessionObject, saveCourseCompletionSessionObject} from './reporting/utils'
+
 const { xss } = require('express-xss-sanitizer')
 
 
 export class ReportingController {
 	router: Router
 	reportService: ReportService
-	dateStartEndCommandFactory: DateStartEndCommandFactory
-	dateStartEndCommandValidator: Validator<DateStartEndCommand>
-	dateStartEndValidator: Validator<DateStartEnd>
 	csrsService: CsrsService
 	organisationalUnitService: OrganisationalUnitService
 
 	constructor(
 		reportService: ReportService,
-		dateStartEndCommandFactory: DateStartEndCommandFactory,
-		dateStartEndCommandValidator: Validator<DateStartEndCommand>,
-		dateStartEndValidator: Validator<DateStartEnd>,
 		csrsService: CsrsService,
 		organisationalUnitService: OrganisationalUnitService
 	) {
 		this.router = Router()
 		this.configureRouterPaths()
 		this.reportService = reportService
-		this.dateStartEndCommandFactory = dateStartEndCommandFactory
-		this.dateStartEndCommandValidator = dateStartEndCommandValidator
-		this.dateStartEndValidator = dateStartEndValidator
 		this.csrsService = csrsService
 		this.organisationalUnitService = organisationalUnitService
 	}
 
 	private configureRouterPaths() {
-		this.router.get('/reporting', xss(), this.getReports())
-
-		this.router.post('/reporting/booking-information', xss(), this.generateReportBookingInformation())
-		this.router.post('/reporting/learner-record', xss(), this.generateReportLearnerRecord())
-
 		this.router.get('/reporting/course-completions/choose-organisation', xss(), this.getChooseOrganisationPage())
 		this.router.post('/reporting/course-completions/choose-organisation', xss(), this.submitOrganisationSelection())
 	}
 
-	getReports() {
-		return async (request: Request, response: Response) => {
-			response.render('page/reporting/index', {
-				placeholder: new PlaceholderDate(),
-			})
-		}
-	}
-
 	getChooseOrganisationPage() {
 		return async (request: Request, response: Response) => {
 			let currentUser = request.user
-			
+
 			if (currentUser && currentUser.isOrganisationReporter() && currentUser.isMVPReporter()) {
 				let organisationChoices = await this.getOrganisationChoicesForUser(currentUser)
 				let userCanAccessMultipleOrganisations: boolean = organisationChoices.typeaheadOrganisations.length > 1
 
-				response.render('page/reporting/choose-organisation', {
+				response.render('page/reporting/courseCompletions/choose-organisation', {
```

Comment:
Should URLs be in this format `page/reporting/course-completions/choose-organisation` instead of this `page/reporting/courseCompletions/choose-organisation`?